src/Avalonia.Svg/AvaloniaModelExtensions.cs

```diff
@@ -117,61 +117,50 @@ public static AM.TextAlignment ToTextAlignment(this SKTextAlign textAlign)
-    {
-        switch (fontStyleWeight)
-        {
-            default:
-            case SKFontStyleWeight.Invisible:
-                // TODO: FontStyleWeight.Invisible
-                throw new NotSupportedException();
-            case SKFontStyleWeight.Thin:
-                return AM.FontWeight.Thin;
-
-            case SKFontStyleWeight.ExtraLight:
-                return AM.FontWeight.ExtraLight;
-
-            case SKFontStyleWeight.Light:
-                return AM.FontWeight.Light;
-
-            case SKFontStyleWeight.Normal:
-                return AM.FontWeight.Normal;
-
-            case SKFontStyleWeight.Medium:
-                return AM.FontWeight.Medium;
-
-            case SKFontStyleWeight.SemiBold:
-                return AM.FontWeight.SemiBold;
-
-            case SKFontStyleWeight.Bold:
-                return AM.FontWeight.Bold;
-
-            case SKFontStyleWeight.ExtraBold:
-                return AM.FontWeight.ExtraBold;
-
-            case SKFontStyleWeight.Black:
-                return AM.FontWeight.Black;
-
-            case SKFontStyleWeight.ExtraBlack:
-                return AM.FontWeight.ExtraBlack;
-        }
-    }
-
-    public static AM.FontStyle ToFontStyle(this SKFontStyleSlant fontStyleSlant)
-    {
-        switch (fontStyleSlant)
-        {
-            default:
-            case SKFontStyleSlant.Upright:
-                // TODO: FontStyleSlant.Upright
-                return AM.FontStyle.Normal;
-            case SKFontStyleSlant.Italic:
-                return AM.FontStyle.Italic;
-
-            case SKFontStyleSlant.Oblique:
-                return AM.FontStyle.Oblique;
-        }
-    }
+    public static AM.FontWeight ToFontWeight(this SKFontStyleWeight fontStyleWeight) =>
+        // The numeric values in these two enumerations have the same scale.
+        // Casting is even encouraged in AM.FontWeight XML documentation.
+        (AM.FontWeight)fontStyleWeight;
+    public static SKFontStyleWeight ToSKFontWeight(this AM.FontWeight fontStyleWeight) =>
+        (SKFontStyleWeight)fontStyleWeight;
+
+    public static AM.FontStretch ToFontStretch(this SKFontStyleWidth fontStyleWidth) => fontStyleWidth switch {
+        SKFontStyleWidth.UltraCondensed => AM.FontStretch.UltraCondensed,
+        SKFontStyleWidth.ExtraCondensed => AM.FontStretch.ExtraCondensed,
+        SKFontStyleWidth.Condensed => AM.FontStretch.Condensed,
+        SKFontStyleWidth.SemiCondensed => AM.FontStretch.SemiCondensed,
+        SKFontStyleWidth.Normal => AM.FontStretch.Normal,
+        SKFontStyleWidth.SemiExpanded => AM.FontStretch.SemiExpanded,
+        SKFontStyleWidth.Expanded => AM.FontStretch.Expanded,
+        SKFontStyleWidth.ExtraExpanded => AM.FontStretch.ExtraExpanded,
+        SKFontStyleWidth.UltraExpanded => AM.FontStretch.UltraExpanded,
+        _ => AM.FontStretch.Normal
+    };
+    public static SKFontStyleWidth ToSKFontStretch(this AM.FontStretch fontStyleWidth) => fontStyleWidth switch {
```

Comment:
Please do not do such refactoring in this PR

Reply:
This is part of AvaloniaAssetLoader implementation